src/pharmpy/tools/covsearch/tool.py

```diff
@@ -479,38 +497,63 @@ def _make_df_steps(best_model: Model, candidates: List[Candidate]):
 
 
 def _make_df_steps_row(
-    models_dict: dict, children_count: Counter, best_model: Model, candidate: Candidate
+    models_dict: dict,
+    children_count: Counter,
+    best_model: Model,
+    candidate: Candidate,
+    index_offset=0,
 ):
     model = candidate.model
     parent_model = models_dict[model.parent_model]
-    reduced_ofv = np.nan if (mfr := parent_model.modelfit_results) is None else mfr.ofv
-    extended_ofv = np.nan if (mfr := model.modelfit_results) is None else mfr.ofv
-    ofv_drop = reduced_ofv - extended_ofv
     if candidate.steps:
         last_step = candidate.steps[-1]
         last_effect = last_step.effect
         parameter, covariate = last_effect.parameter, last_effect.covariate
         extended_state = f'{last_effect.operation} {last_effect.fp}'
         is_backward = isinstance(last_step, BackwardStep)
-        alpha = last_step.alpha
-        extended_significant = lrt_test(
-            parent_model,
-            candidate.model,
-            reduced_ofv,
-            extended_ofv,
-            alpha,
-        )
+        if not is_backward:
+            reduced_ofv = np.nan if (mfr := parent_model.modelfit_results) is None else mfr.ofv
+            extended_ofv = np.nan if (mfr := model.modelfit_results) is None else mfr.ofv
+            alpha = last_step.alpha
+            extended_significant = lrt_test(
+                parent_model,
+                candidate.model,
+                reduced_ofv,
+                extended_ofv,
+                alpha,
+            )
```

Comment:
`ofv_drop` needs to be defined here too right?

Reply:
`ofv_drop` is defined after if statements on line 536